sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -59,11 +59,12 @@ public TextAnalyticsClient(Uri endpoint, TokenCredential credential, TextAnalyti
             Argument.AssertNotNull(credential, nameof(credential));
             Argument.AssertNotNull(options, nameof(options));
 
+            string defaultScope = options.Audience + "/.default";
```

Comment:
```suggestion
            string defaultScope = $"{string.IsNullOrEmpty(options.Audience) ? TextAnalyticsAudience.Azure : options.Audience}/.default";
```

This would require that we leave the `Audience` as nullable and default to <c>null</c> as suggested in the options class.